plugins/katello/3.4/release_notes/release_notes.md

```diff
@@ -4,12 +4,28 @@ title: Release Notes
 version: 3.4
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.4 (Oud bruin) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.4/CHANGELOG.md).
 
 ## Features
 
+#### File Repository Additions
+
+#### Repository Force Sync
+
+#### Content View Force Republish
```

Comment:
On a Content View Version publish or promotion, we were previously regenerating the metadata in the base Library version of the repository. This would happen even if nothing had changed in the repo, and caused pulp to perform extra steps during a Smart Proxy w/ Content sync as the revision number in the repomd.xml file had changed. Now we default to not regenerating the metadata if nothing has changed in the repo. The main benefit of this is that it speeds up Smart Proxy w/ Content syncs.